flopy/utils/gridintersect.py

```diff
@@ -385,18 +417,12 @@ def _rect_grid_to_shape_list(self):
         list :
             list of shapely Polygons
         """
-        return list(self._rect_grid_to_shape_generator())
-
-    def _usg_grid_to_shape_list(self):
-        """internal method, convert unstructred grid to list of shapely
-        polygons.
-
-        Returns
-        -------
-        list
-            list of shapely Polygons
-        """
-        raise NotImplementedError()
+        warnings.warn(
+            "`_rect_grid_to_shape_list()` is deprecated, please"
+            "use `_rect_grid_to_geoms_cellids()` instead.",
+            DeprecationWarning,
+        )
+        return self._rect_grid_to_geoms_cellids()[0]
 
     def _vtx_grid_to_shape_list(self):
         """internal method, list of shapely polygons for vertex grids.
```

Comment:
Reccomend adding a note in the docstrings that indicates that this routine was Deprecated in version 3.3.6.